examples/neon-cloudflare/alchemy.run.ts

```diff
@@ -0,0 +1,60 @@
+/// <reference types="node" />
+
+import alchemy from "alchemy";
+import { NeonProject, NeonBranch, NeonDatabase, NeonRole } from "alchemy/neon";
+import { Worker, Hyperdrive } from "alchemy/cloudflare";
+
+const app = await alchemy("neon-cloudflare-example", {
+  phase: process.argv.includes("--destroy") ? "destroy" : "up",
+});
```

Comment:
set password: process.env.ALCHEMY_PASSWORD

better yet, take a look at alchemy.run.ts in ../cloudflare-worker/alchemy.run.ts